knora/dsplib/utils/xml_upload.py

```diff
@@ -540,21 +606,21 @@ def _upload_stashed_xml_texts(
         nonapplied_xml_texts: the xml texts that could not be uploaded
     """
 
-    print('Upload the stashed XML texts...')
+    logger.info("Upload the stashed XML texts...")
     for resource, link_props in stashed_xml_texts.copy().items():
         if resource.id not in id2iri_mapping:
             # resource could not be uploaded to DSP, so the stash cannot be uploaded either
             continue
         res_iri = id2iri_mapping[resource.id]
         try:
             existing_resource = try_network_action(
-                action=lambda: con.get(path=f'/v2/resources/{quote_plus(res_iri)}'),
-                failure_msg=f'  ERROR while retrieving resource "{resource.id}" from DSP server.'
+                action=lambda: con.get(path=f"/v2/resources/{quote_plus(res_iri)}"),
+                failure_msg=f"  ERROR while retrieving resource '{resource.id}' from DSP server."
             )
         except BaseError as err:
-            print(err.message)
+            logger.error(err.message)
             continue
-        print(f'  Upload XML text(s) of resource "{resource.id}"...')
+        logger.info(f"  Upload XML text(s) of resource '{resource.id}'...")
```

Comment:
Is it still necessary to have the spaces in front of the log message(s) (also in other locations)?